go/cmd/server/server_test.go

```diff
@@ -137,4 +166,139 @@ func TestGetOnlineFeaturesSqlite(t *testing.T) {
 	assert.True(t, reflect.DeepEqual(response.Results[3].Values, expectedAvgDailyTripsValues))
 
 	assert.True(t, reflect.DeepEqual(response.Metadata.FeatureNames.Val, expectedFeatureNamesResp))
+	time.Sleep(1 * time.Second)
+}
+
+func TestGetOnlineFeaturesSqliteWithLogging(t *testing.T) {
+	ctx := context.Background()
+	// Pregenerated using `feast init`.
+	dir := "."
+	err := test.SetupFeatureRepo(dir)
+	assert.Nil(t, err)
+	defer test.CleanUpRepo(dir)
+	client, closer := getClient(ctx, dir, true)
+	defer closer()
+	entities := make(map[string]*types.RepeatedValue)
+	entities["driver_id"] = &types.RepeatedValue{
+		Val: []*types.Value{
+			{Val: &types.Value_Int64Val{Int64Val: 1001}},
+			{Val: &types.Value_Int64Val{Int64Val: 1003}},
+			{Val: &types.Value_Int64Val{Int64Val: 1005}},
+		},
+	}
+
+	request := &serving.GetOnlineFeaturesRequest{
+		Kind: &serving.GetOnlineFeaturesRequest_FeatureService{
+			FeatureService: "test_service",
+		},
+		Entities: entities,
+	}
+	response, err := client.GetOnlineFeatures(ctx, request)
+	// Wait for logger to flush.
+	time.Sleep(1 * time.Second)
```

Comment:
`assert.Eventually` is better way to wait for smth

Reply:
Fixed.